.jest/setEnvVars.ts

```diff
@@ -0,0 +1,3 @@
+process.env.SERVICE_EMAIL_ADDRESS = 'mock-service-address@email.co.uk';
```

Comment:
Little bit confused whats going on here with the mock email address... wont this make it currently not work?